app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -11,11 +11,12 @@ def initialize
         @validate_resource_restrictions_class = Authentication::ResourceRestrictions::ValidateResourceRestrictions
       end
 
-      def create_authentication_parameters(authenticator_input)
-        @authentication_parameters = @authentication_parameters_class.new(authenticator_input)
-        @authentication_parameters.jwt_token = @extract_token_from_credentials.call(
-          credentials: authenticator_input.request.read
+      def authentication_parameters(authenticator_input)
+        authentication_parameters = @authentication_parameters_class.new(authenticator_input)
+        authentication_parameters.jwt_token = @extract_token_from_credentials.call(
+          credentials: authenticator_input.request.body.read
         )
+        authentication_parameters
```

Comment:
not sure where the conversation about this went but i'd like to know if you have an issue to refactor this so that `jwt_token` is given in the constructor of `authentication_parameters_class` instead of setting it as an `attr_accessor`.

Reply:
https://github.com/cyberark/conjur/pull/2199#discussion_r637855005